packages/web/hooks/one-click-trading/use-one-click-trading-info.ts

```diff
@@ -1,28 +1,67 @@
 import type { OneClickTradingInfo } from "@osmosis-labs/stores";
 import { unixNanoSecondsToSeconds } from "@osmosis-labs/utils";
 import dayjs from "dayjs";
-import { useEffect, useState } from "react";
+import { useCallback, useEffect, useState } from "react";
 import { useAsync } from "react-use";
 
 import { useTranslation } from "~/hooks/language";
 import { useStore } from "~/stores";
 
-export const useOneClickTradingInfo = ({
+/**
+ * This hook manages and provides information about the one-click trading session.
+ *
+ * This hook encapsulates the logic for fetching one-click trading session information,
+ * determining if the session is enabled and if it has expired. It also provides utility
+ * functions to calculate the remaining time for the session and the total session time.
+ *
+ * @param {Object} params - The parameters object.
```

Comment:
NIT: would prefer to avoid specifying types in js-doc, would prefer just typescript.